service-tests/github.js

```diff
@@ -9,13 +9,50 @@ const {
   isFormattedDate,
   isVPlusDottedVersionAtLeastOne
 } = require('./helpers/validators');
+const colorscheme = require('../lib/colorscheme.json');
 
 const t = new ServiceTester({ id: 'github', title: 'Github' });
 module.exports = t;
-
-t.create('License')
-  .get('/license/badges/shields.json')
-  .expectJSONTypes(Joi.object().keys({ name: 'license', value: Joi.string() }));
+const colorsB = Object.assign({}, ...Object.keys(colorscheme).map(color => ({ [color]: colorscheme[color].colorB })));
```

Comment:
It's probably worth installing [`lodash.mapvalues`](https://lodash.com/docs/4.17.4#mapValues) which would make this clearer:
```js
const mapValues = require('lodash.mapvalues');
const colorsB = mapValues(colorscheme, 'colorB');
```

Reply:
I will apply all this changes to npm tests as well.